src/platform_impl/linux/x11/window.rs

```diff
@@ -91,6 +92,7 @@ impl SharedState {
             max_inner_size: None,
             resize_increments: None,
             base_size: None,
+            has_focus: true,
```

Comment:
Shouldn't this be `false`?

Reply:
doesn't the window start initially with keyboard focus?

Reply:
On macOS, the window is initially not focused (and `window.has_focus()` returns `false`), but it receives a `WindowEvent::Focused` shortly after launching

Reply:
This only affects x11, on macOS we query the OS directly. The reason I made x11 start with `true` is because Windows also starts with `true` and I think x11 is very similar to Windows in this matter, but I haven't had the chance to test x11 (I may test it later today).